kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -244,6 +248,11 @@ private static List<DynamicTest> testsForDir(List<DesiredFn<KafkaProxy, ?>> depe
                                 assertThat(Files.exists(expectedFile)).describedAs(expectedFile + " does not exist").isTrue();
                                 var expected = loadExpected(expectedFile, resourceType);
                                 assertSameYaml(actualResource, expected);
+                                if (actualResource instanceof ConfigMap cm && cm.getData().containsKey("proxy-config.yaml")) {
```

Comment:
a bit crusty to sniff for this config map entry, but it feels like a good place to check we are emitting something parseable